sdks/typescript/src/step.ts

```diff
@@ -296,6 +289,14 @@ export class Context<T, K = {}> {
     return this.action.workflowRunId;
   }
 
+  /**
+   * Gets the ID of the current workflow run.
+   * @returns The workflow run ID.
+   */
+  taskRunId(): string {
```

Comment:
nit: potentially confusing mix of `workflow run id` and `task run id` here

Reply:
good catch